.github/workflows/comprehensive.yml

```diff
@@ -0,0 +1,442 @@
+      run: |
+        echo " --------------------------- remove old packages ------------------------------------------------------------------------------------------------------ "
+        sudo apt-get remove docker* containerd* || true
+        echo " --------------------------- install containerd.io ---------------------------------------------------------------------------------------------------- "
+        sudo apt-get install containerd.io
+        echo " --------------------------- install docker.io -------------------------------------------------------------------------------------------------------- "
+        echo " --------------------------- install docker-compose --------------------------------------------------------------------------------------------------- "
+        sudo apt-get install docker-compose
+
+    - name: INIT - reconfigure docker and restart its daemon
+      run: |
+        sudo systemctl unmask docker
+        sudo groupadd docker || true
+        sudo gpasswd -a username docker || true
+        sudo service docker restart || true
+
+    - name: INIT - uninstall old Node JS if any
+      run: |
+        sudo npm cache clean -f || true &> /dev/null
+        sudo apt-get remove nodejs npm node -y || true &> /dev/null
+        sudo apt-get purge nodejs -y || true &> /dev/null
+        sudo apt-get autoremove -y || true &> /dev/null
+        sudo rm -rf /usr/bin/node || true &> /dev/null
+        sudo rm -rf /usr/include/node || true &> /dev/null
+        sudo rm -rf /usr/lib/node_modules || true &> /dev/null
+        sudo rm -rf /usr/local/bin/npm  || true &> /dev/null
+        sudo rm -rf /usr/local/share/man/man1/node* || true &> /dev/null
+        sudo rm -rf /usr/local/lib/dtrace/node.d || true &> /dev/null
+        sudo rm -rf ~/.npm || true &> /dev/null
+        sudo rm -rf ~/.node-gyp || true &> /dev/null
+        sudo rm -rf /opt/local/bin/node || true &> /dev/null
+        sudo rm -rf /opt/local/include/node || true &> /dev/null
+        sudo rm -rf /opt/local/lib/node_modules || true &> /dev/null
+        sudo rm -rf /usr/local/lib/node* || true &> /dev/null
+        sudo rm -rf /usr/local/include/node* || true &> /dev/null
+        sudo rm -rf /usr/local/bin/node* || true &> /dev/null
+
+    - name: INIT - install Node JS
+      run: |
+        curl -sL https://deb.nodesource.com/setup_16.x | sudo bash -
+        sudo apt-get install -y nodejs
+        sudo ln -s /usr/bin/node /usr/local/bin/node
+        sudo npm install --global npm
+        sudo npm install --global color-support
+        sudo npm install --global yarn
+        sudo npm install --global node-gyp
+
+    - name: INIT - version checks of Node JS and its utilities
+      run: |
+        which node
+        node --version
+        which npm
+        npm --version
+        which yarn
+        yarn --version
+        which node-gyp
+        node-gyp --version
+
+    - name: INIT - Download comprehensive-test
```

Comment:
There is a github action for doing this:
`actions/checkout`

Reply:
Same as previous.